subprojects/language-java/src/main/java/org/gradle/external/javadoc/JavadocOfflineLink.java

```diff
@@ -16,12 +16,13 @@
 
 package org.gradle.external.javadoc;
 
+import java.io.Serializable;
 
 /**
  * This class is used to hold the information that can be provided to the javadoc executable via the -linkoffline
  * option.
  */
-public class JavadocOfflineLink {
+public class JavadocOfflineLink implements Serializable {
```

Comment:
Good point (though we shouldn't rely on people doing this, and use the serialized value instead).